microsite/data/plugins/kubernetes-provider.yaml

```
@@ -0,0 +1,10 @@
+---
+title: Kubernetes Provider
```

```suggestion
title: Kubernetes Entity Provider
```

Slight tweak to alighn with the other entity providers and avoid confusion